gpu-operator/dra-driver.rst

```diff
@@ -0,0 +1,504 @@
+About the NVIDIA GPU DRA Driver
+*******************************
+
+The NVIDIA GPU DRA Driver leverages the Kubernetes Dynamic Resource Allocation (DRA) API to support NVIDIA IMEX channels available in GH200 and GB200 GPUs.
+The NVIDIA GPU DRA Driver creates and manages IMEX channels through the creation of a ComputeDomain custom resource. You use this custom resource to define your resource templates, and then reference the templates within your workload specs. The GPU DRA Driver will handle creating the cooresponding IMEX channels for you. 
+
+An IMEX channel is a construct that allows a set of GPUs to directly read and write each other's memory over a high-bandwidth NVLink. 
+The NVLink connection may either be directly between GPUs on the same node or between GPUs on separate nodes connected by an NVSwitch. 
+Once an IMEX channel has been established for a set of GPUs, they are free to read and write each other's memory via extensions to the CUDA memory call APIs.
+
+An IMEX channel is a resource that spans multiple nodes. 
+GH200 and GB200 systems are designed specifically to leverage the use of IMEX channels to turn a rack of GPU machines, each with a small number of GPUs, into a giant supercomputer with up to 72 GPUs communicating at full NVLink bandwidth.
+This allows you to get the most use out of your available GPUs without any additional latency burdens.  
+
+Kubernetes DRA, available as beta from Kubernetes v1.32, is an API for requesting and sharing resources between pods and containers inside a pod. 
+This feature treats specialized hardware as a definable and reusable object and provides the necessary primitives to support cross-node resources such as IMEX channels. 
+NVIDIA GPU DRA Driver uses DRA features to define IMEX channel resources that can be managed by Kubernetes.
+
+Refer to the `Kubernetes DRA documentation <https://kubernetes.io/docs/concepts/scheduling-eviction/dynamic-resource-allocation/>`__ for details on this feature. 
+
+*********************************
+Install the NVIDIA GPU DRA Driver 
+*********************************
+
+The GPU DRA Driver is an addiitonal component that can be installed after you've installed the GPU Operator on your clsuter.
+
+Prerequisites
+=============
+
+- GH200 and GB200 GPUs with Mulit-Node NVLink connections between GPUs.
+
+- Kubernetes v1.32 multi-node cluster with the DynamitcResourceAllocation feature gate enabled. 
+
+  The following is a sample for enabling DRA feature gates. 
+  Refer to the Kubernetes documentation for full details on `enabling DRA on your cluster <https://kubernetes.io/docs/concepts/scheduling-eviction/dynamic-resource-allocation/#enabling-dynamic-resource-allocation>`__.
+
+  .. literalinclude:: ./manifests/input/kubeadm-init-config.yaml
+    :language: yaml
+    :caption: Sample Kubeadm Init Config with DRA Feature Gates Enabled
+
+- The NVIDIA GPU Operator v25.3.0 or later installed with CDI enabled on all nodes and NVIDIA GPU Driver 565 or later.

```

Comment:
The v565 driver is only for GH200 / GB200 systems where MNNVL is supported. For other systems older driver versions should be sufficient.

Reply:
This entire page is specific to documenting MNNVL. So I think it is okay to mention the minimum driver version that supports MNNVL.